.circleci/config.yml

```diff
@@ -0,0 +1,46 @@
+version: 2
+
+jobs:
+  build:
+    machine:
+        enabled: true
+    steps:
+      - checkout
+      - run:
+          name: Install Nix
+          command: |
+            sudo mkdir -p /nix
+            sudo chown circleci /nix
+            bash <(curl https://nixos.org/nix/install)
+            echo '. /home/circleci/.nix-profile/etc/profile.d/nix.sh' >> $BASH_ENV
+            sudo mkdir -p /etc/nix
+
+            # Enable sandbox
+            echo "build-use-sandbox = true" | sudo tee -a /etc/nix/nix.conf
+            echo "substituters = https://cache.nixos.org https://static-haskell-nix.cachix.org" \
+                | sudo tee -a /etc/nix/nix.conf
+            echo "trusted-public-keys = cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= static-haskell-nix.cachix.org0:Q17HawmAwaM1/BfIxaEDKAxwTOyRVhPG5Ji9K3+FvUU=" \
+                | sudo tee -a /etc/nix/nix.conf
+
+      - run:
+          name: Install cachix
+          command: |
+            nix-env -iA cachix -f https://cachix.org/api/v1/install
+
+      - run:
+          name: Run cachix
+          command: |
+            cachix push deckgo --watch-store
```

Comment:
Should this be `static-haskell-nix`?

Don't we have to authenticate it somehow so that it can upload?

Reply:
Definitely should be static-haskell-nix, copy paste error. The authentication happens through the token passed as an env var, See commit message

Reply:
Makes sense, thanks!